### ISSUE TITLE
Possible Incorrect Default Path

### ISSUE THREAD

**Issue by Mike-EEE** [dotnet/docs#5309]
Checking out this command here.  the `-o` states that `./bin/[configuration]/[framework]/` is the default output.  While this is created, I am also seeing a `./bin/[configuration]/[framework]/publish` that has all of the deployed (published) artifacts.  I am also using 2.1 RC1 with Visual Studio 2017 15.7.  Not sure if it's because of this, but thought I would mention it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 211afdc7-c796-202d-239d-823f2a499dff
* Version Independent ID: 9439897e-dcb3-bc2e-5cae-35c1ca2e0681
* Content: [dotnet publish command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish?tabs=netcore2x)
* Content Source: [docs/core/tools/dotnet-publish.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-publish.md)
* Service: **dotnet-core**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

**Comment by mairaw** [comment]
I'll double check @Mike-EEE but CLI and Visual Studio might have different behaviors.

**Comment by Mike-EEE** [comment]
Yeah I am still piecing all the pieces together myself, @mairaw.  To clarify on my side, what I meant was that I have Visual Studio installed and did not know if that was impacting anything.  I am using the `dotnet publish` command outside of Visual Studio in a solution directory that was created by Visual Studio and it is producing this `publish` folder.

Additionally, at one point I was using the built-in Visual Studio publish functionality and was unclear if that was contributing to this, but that does not appear to be the case as I have created several Visual Studio projects since then without any profiles and this `publish` folder still gets created.
